server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```diff
@@ -258,16 +258,16 @@ data:
             "logSavedOpsTimeIntervalMs": {{ .Values.scriptorium.logSavedOpsTimeIntervalMs }},
             "opsCountTelemetryEnabled": {{ .Values.scriptorium.opsCountTelemetryEnabled }},
             "circuitBreakerEnabled": {{ .Values.scriptorium.circuitBreakerEnabled }},
-            "circuitBreakerOptions": {
-                "errorThresholdPercentage": {{ .Values.scriptorium.circuitBreakerOptions.errorThresholdPercentage }},
-                "resetTimeout": {{ .Values.scriptorium.circuitBreakerOptions.resetTimeout }},
-                "timeout": {{ .Values.scriptorium.circuitBreakerOptions.timeout }},
-                "rollingCountTimeout": {{ .Values.scriptorium.circuitBreakerOptions.rollingCountTimeout }},
-                "rollingCountBuckets": {{ .Values.scriptorium.circuitBreakerOptions.rollingCountBuckets }},
-                "fallbackToRestartTimeoutMs": {{ .Values.scriptorium.circuitBreakerOptions.fallbackToRestartTimeoutMs }},
+            "circuitBreakerOptionsDb": {
```

Comment:
Instead of using a new node, how about using `circuitBreakerOptions` array, and in the array specify used for `db`? Then code filter on db for db?